src/main/java/jetbrains/buildServer/auth/oauth/ConfigKey.java

```diff
@@ -11,6 +11,6 @@
     scope,
     hideLoginForm,
     allowInsecureHttps,
-    emailDomain,
+    emailDomains,
```

Comment:
I don’t like that we don’t keep old key, as it will break when someone is using it already, can you reuse existing key.